html/captive-portal/templates/wireless-profile.xml

```diff
@@ -6,55 +6,145 @@
         <key>PayloadContent</key>
         <array>
                 <dict>
+                        [% IF provisioner.eap_type != "" and provisioner.security_type != "none" %]
+                        <key>AutoJoin</key>
+                        <true/>
                         <key>EAPClientConfiguration</key>
                         <dict>
                                 <key>AcceptEAPTypes</key>
                                 <array>
-                                        <integer>25</integer>
+                                        <integer>[% provisioner.eap_type %]</integer>
+                                </array>
+                                <key>PayloadCertificateAnchorUUID</key>
+                                <array>
+                                        <string>96148a46-d87a-11e4-b9d6-1681e6b88ec1</string>
```

Comment:
Are we sure this value fits when running on a customer setup. This seems pretty specific to your setup